test/regression/expected/duckdb_recycle.out

```diff
@@ -54,4 +49,24 @@ EXPLAIN SELECT count(*) FROM ta;
  
 (19 rows)
 
+-- Not allowed in a transaction
+BEGIN;
+CALL duckdb.recycle_ddb();
+ERROR:  duckdb.recycle_ddb() cannot run inside a transaction block
+END;
+-- Nor in a function
+CREATE OR REPLACE FUNCTION f() RETURNS void
+    LANGUAGE plpgsql
+    RETURNS NULL ON NULL INPUT
+    AS
+$$
+BEGIN
+    CALL duckdb.recycle_ddb();
```

Comment:
I changed this to use the `CALL` syntax, so we can easily detect when `recycle_ddb` is not run at the top level (like shown here).